PowerShell/ScubaGear/Modules/Providers/ExportAADProvider.psm1

```diff
@@ -89,6 +89,9 @@ function Export-AADProvider {
     }
     $ServicePlans = ConvertTo-Json -Depth 3 @($ServicePlans)
 
+    #Obtains license information for tenant
+    $LicenseInfo = $Tracker.TryCommand("Get-MgBetaSubscribedSku") | Select-Object -Property Sku*, ConsumedUnits, PrepaidUnits | ConvertTo-Json -Depth 3
```

Comment:
Yes, there are add-ons such as AAD P2, but we already obtain the Service Plan ID. We can get the ServicePlans within the SKUs but it will substantially lengthen the JSON since there are over 70 in tenants such as E5. Also, in testing so far all of the service plans fall under the SKU for the tenant and the Flow_Free has some by default.

Reply:
Ok, I see what you mean in AAD.additionaldata.  We can leave it out for now.   Can you send me (via slack or email) some examples of what this data will look like for the different tenants we have.  In particular, I'm curious about the situation of G3 + aad_p2 add on (which unfortunately we might not actually have in a tenant).

Reply:
I have sent you via slack some examples of what the data would look like for 2 of our tenants. If you would like more of different examples please let me know.